dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1177,6 +1146,38 @@ private void processWesCommands(final List<String> args) {
 
     }
 
+    /**
+     * This will aggregate the WES request URI and credentials into a single object for use down the line
+     * @param args The commaand line arguments
+     */
+    private void aggregateWesRequestData(final List<String> args) {
+
+        // Get the config file to see if credentials are there
+        INIConfiguration config = Utilities.parseConfig(this.getConfigFile());
+        SubnodeConfiguration configSubNode = null;
+        try {
+            configSubNode = config.getSection("WES");
+        } catch (Exception e) {
+            out("Could not get WES section from config file");
+        }
+
+        // Attempt to find the WES URI
+        final String wesUri = optVal(args, "--wes-url", null);
+        final String wesEndpointUrl = ObjectUtils.firstNonNull(wesUri, Objects.requireNonNull(configSubNode).getString("url"));
+
+        // Depending on the endpoint (AWS/non-AWS) we need to look for a different set of credentials
+        final boolean isAwsWes = flagPresent(args, "--aws");
```

Comment:
This only applies to WES, and all other wes-specific params start with `--wes`.

Also, if you have both `--wes-auth` and `--aws`, `--aws` wins out (currently, the way the code is). I'd been thinking of something like `--wes-auth aws`, but I don't know if that's good practice either.

I don't know the answer here, just brainstorming.